src/index.js

```diff
@@ -362,7 +362,7 @@ class Superlogin extends EventEmitter2 {
 			})
 			.catch(err => {
 				this._onLogout(msg || 'Logged out');
-				if (err.response.data.status !== 401) {
+				if (!err.response || (err.response && err.response.data.status !== 401)) {
```

Comment:
No need for the double check. The following should be enough
`(!err.response ||  err.response.data.status !== 401)`